readthedocs/api/v3/serializers.py

```diff
@@ -545,6 +545,9 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
+        user = self.context['request'].user
+        self.fields['child'].queryset = user.projects.all()
```

Comment:
This is better than what we have now and seems enough to me.

I'd add a TODO comment here saying that these projects can be filtered using the restrictions that we already known: "project is already a subproject", "project is the same as subproject", etc. I don't think have to do this in this PR. Just the TODO comment --we can easily achieve this once we have an isolated function for this.

Reply:
Done! :)

Reply:
:tada: